test/unit/spec/EnrollU2F_spec.js

```diff
@@ -103,7 +113,46 @@ function (Okta,
     });
 
     Expect.describe('Enroll factor', function () {
+      itp('shows error if wrong browser', function () {
+        mockChrome(false);
+        mockFirefox(false);
+
+        return setup().then(function (test) {
+          expect(test.form.errorHtml()).toHaveLength(1);
+        });
+      });
+
+      itp('shows error if Firefox without extension', function () {
+        mockChrome(false);
+        mockFirefox(true);
+        delete window.u2f;
+
+        return setup().then(function (test) {
+          expect(test.form.errorHtml()).toHaveLength(1);
```

Comment:
similar to my previous nit, this could be
```
expect(test.form.errorHtml().indexOf('install the Firefox U2F browser extension before proceeding') > -1).toBe(true);
```